.circleci/config.yml

```diff
@@ -1,23 +1,15 @@
 version: 2.1
 
-environment: &base-env
-    NPM_CONFIG_PROGRESS: 'false'
-    NPM_CONFIG_LOGLEVEL: warn
-    
-executors:
-  node-container:
-    docker:
-      - image: node:12
-        environment: *base-env
-
-  node-container-next-lts:
-    docker:
-      - image: node:14
-        environment: *base-env
+orbs:
+  node: circleci/node@4.1.0
 
 jobs:
   dependency-check:
-    executor: node-container
+    docker:
+       - image: node:12
+         environment:
+            NPM_CONFIG_PROGRESS: 'false'
+            NPM_CONFIG_LOGLEVEL: warn
```

Comment:
If you want to use the orb, I would sill attach this environment to the orb and use it across the board. Otherwise we get inconsistent result among different runs

Reply:
What do you mean with "inconsistent"? This only changes the logs, right?